com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+
+            if (mlapiModules != null)
+            {
+                bool hasChanged = UpdateOrCreateModule(ref mlapiModules, RPCModuleName, CreateRPCCounters);
+                hasChanged = hasChanged && UpdateOrCreateModule(ref mlapiModules, OperationModuleName, CreateOperationsCounters);
+                hasChanged = hasChanged && UpdateOrCreateModule(ref mlapiModules, MessageModuleName, CreateMessagesCounters);
+                if (hasChanged)
+                {
+                    EditorPrefs.SetString("ProfilerWindow.DynamicModules", JsonUtility.ToJson(mlapiModules));
+                }
+            }
+
+            MLAPI.NetworkingManager.OnPerformanceDataEvent += OnPerformanceTickData;
+#endif
+        }
+
+        static void OnPerformanceTickData(PerformanceTickData tickData)
+        {
+
+#if UNITY_2020_2_OR_NEWER
+            //Operations
+            ProfilerInfo.ConnectionsCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfConnections.ToString());
+            ProfilerInfo.TickRateCounterValue.Value += tickData.GetData(ProfilerConstants.ReceiveTickRate.ToString());
+
+            //Messages
+            ProfilerInfo.NamedMessagesCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfNamedMessages.ToString());
+            ProfilerInfo.UnnamedMessagesCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfUnnamedMessages.ToString());
+            ProfilerInfo.BytesSentCounterValue.Value += tickData.GetData(ProfilerConstants.NumberBytesSent.ToString());
+            ProfilerInfo.BytesReceivedCounterValue.Value += tickData.GetData(ProfilerConstants.NumberBytesReceived.ToString());
+            ProfilerInfo.NetworkVarsCounterValue.Value += tickData.GetData(ProfilerConstants.NumberNetworkVarsReceived.ToString());
+
+            //RPCs
+            ProfilerInfo.RPCsSentCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCsSent.ToString());
+            ProfilerInfo.RPCsReceivedCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCsReceived.ToString());
+            ProfilerInfo.RPCBatchesSentCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCBatchesSent.ToString());
+            ProfilerInfo.RPCBatchesReceivedCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCBatchesReceived.ToString());
+            ProfilerInfo.RPCBatchesReceivedCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCBatchesReceived.ToString());
+            ProfilerInfo.RPCQueueProcessedCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCQueueProcessed.ToString());
+            ProfilerInfo.RPCsInQueueSizeCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCsInQueueSize.ToString());
+            ProfilerInfo.RPCsOutQueueSizeCounterValue.Value += tickData.GetData(ProfilerConstants.NumberOfRPCsOutQueueSize.ToString());
+#endif
+        }
+
+        // private static List<NetworkCounterData> GetCustomMessageCounters()
```

Comment:
With this much code commented out, can you surround with an #ifdef and put a comment explaining what this code might be for in the future?  (or delete)

Reply:
please delete the commented code

Reply:
Commented code has been deleted